model-bundler/bundle-api/model/bundle.smithy

```diff
@@ -0,0 +1,25 @@
+$version: "2"
+
+namespace software.amazon.smithy.modelbundle.api
+
+structure Bundle {
+    @required
+    configType: String
+
+    @required
+    serviceName: String
+
+    @required
+    config: Document
+}
+
+structure SimpleAwsConfig {
```

Comment:
This should probably be outside this package, in an aws specific one.

Reply:
This doesn't need to be in here anyway, it's unused